### DISCUSSION TITLE
Add custom fonts

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#292] [Feature Suggestions]
I would like the option to click "custom font", and select a custom TTF or OTF file font.

**Comment by byemc** [comment]
> I would like the option to click "custom font", and select a custom TTF or OTF file font.

Shouldn't this let you pick from Google's font library as well?

**Reply by ghost** [reply]
i was thinking using a .ttf file

**Comment by aurkaxi** [comment]
+1

**Comment by ghost** [comment]
I was about to suggest an option to use your system/browser's default font, after i changed mine and noticed it doesn't affect Revolt. This sounds better.
